test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -0,0 +1,245 @@
+
+    framework = test_utils.get_framework_from_image_uri(training)
+
+    package_list_cmd = "conda list" if "pytorch" in framework else "pip freeze"
+
+    test_utils.run_cmd_on_container(container_name, ctx, f"{package_list_cmd} | grep -i imageio")
+    test_utils.run_cmd_on_container(
+        container_name, ctx, "import imageio; print(imageio.__version__)", executable="python"
+    )
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("opencv")
+def test_opencv(training):
+    """
+    Verify that cv2 is installed in the Training DLC image
+
+    :param training: training ECR image URI
+    """
+    ctx = Context()
+    container_name = test_utils.get_container_name("opencv", training)
+    test_utils.start_container(container_name, training, ctx)
+
+    framework = test_utils.get_framework_from_image_uri(training)
+
+    package_list_cmd = "conda list" if "pytorch" in framework else "pip freeze"
+
+    test_utils.run_cmd_on_container(container_name, ctx, f"{package_list_cmd} | grep -i opencv")
+    test_utils.run_cmd_on_container(container_name, ctx, "import cv2; print(cv2.__version__)", executable="python")
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("pandas")
+def test_pandas(image):
+    """
+    It's possible that in newer python versions, we may have issues with installing pandas due to lack of presence
+    of the bz2 module in py3 containers. This is a sanity test to ensure that pandas import works properly in all
+    containers.
+
+    :param image: ECR image URI
+    """
+    ctx = Context()
+    container_name = test_utils.get_container_name("pandas", image)
+    test_utils.start_container(container_name, image, ctx)
+
+    # Make sure we can install pandas, do not fail right away if there are pip check issues
+    test_utils.run_cmd_on_container(container_name, ctx, "pip install pandas", warn=True)
+
+    pandas_import_output = test_utils.run_cmd_on_container(container_name, ctx, "import pandas", executable="python")
+
+    assert (
+        not pandas_import_output.stdout.strip()
+    ), f"Expected no output when importing pandas, but got  {pandas_import_output.stdout}"
+
+    # Simple import test to ensure we do not get a bz2 module import failure
+    test_utils.run_cmd_on_container(
+        container_name, ctx, "import pandas; print(pandas.__version__)", executable="python"
+    )
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("plotly")
+def test_plotly(training):
+    """
+    Verify that plotly is installed in the Training DLC image
+
+    :param training: ECR image URI
+    """
+    ctx = Context()
+    container_name = test_utils.get_container_name("plotly", training)
+    test_utils.start_container(container_name, training, ctx)
+
+    framework = test_utils.get_framework_from_image_uri(training)
+
+    package_list_cmd = "conda list" if "pytorch" in framework else "pip freeze"
+
+    test_utils.run_cmd_on_container(container_name, ctx, f"{package_list_cmd} | grep -i plotly")
```

Comment:
can we club together all packages import in single function (probably make a list of packages and loop through to check their imports), its seem repetitive to have func for every package with similar code.